### PR TITLE
Developers Session

### PR DESCRIPTION
**Pull request by dz0ny**
Add Developers Session agenda and outline.

### CODE REVIEW

File: 2_Operations/work-process.md [hunk 1]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
 
 The main purpose of the alignment meeting is to adjust our compasses so that we will select the correct user stories in the sprint planning meeting coming the next day. We know we are paddling hard, but are we all paddling in the right direction?
 
+## Developers Session
+
+We are a remote company and as such sharing of ideas and what is learned is hardened by the fact that we don't have the time or space to do that effectively.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `what is learned` -> `lessons learned` 
`is harderend` -> `is impaired`

File: 2_Operations/work-process.md [hunk 2]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
 
 The main purpose of the alignment meeting is to adjust our compasses so that we will select the correct user stories in the sprint planning meeting coming the next day. We know we are paddling hard, but are we all paddling in the right direction?
 
+## Developers Session
+
+We are a remote company and as such sharing of ideas and what is learned is hardened by the fact that we don't have the time or space to do that effectively. 
+
+In the long term, this is hurting everyone and as such `Developers Sessions` are a way to discuss ideas, questions that might have arisen during retrospective or the sprint. Followed by a short lightning talk(s), which can be impromptu or declared beforehand so others can join.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `ideas, questions` -> `ideas and questions`

File: 2_Operations/work-process.md [hunk 3]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
 
+## Developers Session
+
+We are a remote company and as such sharing of ideas and what is learned is hardened by the fact that we don't have the time or space to do that effectively. 
+
+In the long term, this is hurting everyone and as such `Developers Sessions` are a way to discuss ideas, questions that might have arisen during retrospective or the sprint. Followed by a short lightning talk(s), which can be impromptu or declared beforehand so others can join. 
+
+You can learn something from every single person in the world, no matter what their opinion on any topic is. We also believe that this is a big part of open source and our company, being able to learn and collaborate with people from all over the world who have a wide variety of different opinions on how to do things. On the other hand, a lightning talk can be a perfect way for testing how people react to a talk that you are going to have at next conference or just getting rid of that presentation fear in front of people.
+
+In order for things not to explode gathering is limited to 30min plus any lightning talks to 5min.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `In order for things to not explode, the Session is limited to 30 minutes plus any lightning talks to 5min.`

File: 2_Operations/work-process.md [hunk 4]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+
+### Agenda
+
+As attendee ask yourself:
```
Comment (zupo) [inline 2_Operations/work-process.md]: As an attendee, ask yourself

File: 2_Operations/work-process.md [hunk 5]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+
+- Did I have a problem with understanding of implementation I saw during the sprint? 
+- Was I annoyed by a particular workflow or something that I think it's wasting my time?
```
Comment (zupo) [inline 2_Operations/work-process.md]: `it's` -> `is`

File: 2_Operations/work-process.md [hunk 6]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- Did I found a handy tool/method/module that I think we should use?
```
Comment (zupo) [inline 2_Operations/work-process.md]: `Did I found` -> `Did I find`

File: 2_Operations/work-process.md [hunk 7]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- Do I have an idea about implementation we will have in next sprint, and I want to discuss it?
```
Comment (zupo) [inline 2_Operations/work-process.md]: `the next sprint`

File: 2_Operations/work-process.md [hunk 8]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- Have I seen the great technical talk, I want to share it so we can discuss it next time?
```
Comment (zupo) [inline 2_Operations/work-process.md]: `Have I seen a great technical talk and I want to share it so we can discuss it the next time?`

File: 2_Operations/work-process.md [hunk 9]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+
+As attendee ask yourself:
+
+- Did I have a problem with understanding of implementation I saw during the sprint? 
+- Was I annoyed by a particular workflow or something that I think it's wasting my time?
+- Did I found a handy tool/method/module that I think we should use?
+- Do I have an idea about implementation we will have in next sprint, and I want to discuss it?
+- Have I seen the great technical talk, I want to share it so we can discuss it next time? 
+
+As curator:
```
Comment (zupo) [inline 2_Operations/work-process.md]: `As a curator:`

File: 2_Operations/work-process.md [hunk 10]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+
+- Announce in #operations Zoom session to join into
```
Comment (zupo) [inline 2_Operations/work-process.md]: Can we use the niteo zoom room?

File: 2_Operations/work-process.md [hunk 11]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- As curator meet and greet everyone, especially first-time joiners.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `As a curator, ...`

File: 2_Operations/work-process.md [hunk 12]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- Was I annoyed by a particular workflow or something that I think it's wasting my time?
+- Did I found a handy tool/method/module that I think we should use?
+- Do I have an idea about implementation we will have in next sprint, and I want to discuss it?
+- Have I seen the great technical talk, I want to share it so we can discuss it next time? 
+
+As curator:
+
+- Announce in #operations Zoom session to join into
+- As curator meet and greet everyone, especially first-time joiners. 
+- If you hear crickets, start rolling the ball with a question, an anecdote or funny story.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `or funny story` -> `or a funny story`

File: 2_Operations/work-process.md [hunk 13]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- Make sure no one is talking the whole 30min, everyone should have an opinion!!!
```
Comment (zupo) [inline 2_Operations/work-process.md]: `30min` -> `30 minutes`
`!!!` -> `!`

File: 2_Operations/work-process.md [hunk 14]
```diff
@@ -42,6 +43,34 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
+- After 30min or less announce that there is time for Lightning talks.
```
Comment (zupo) [inline 2_Operations/work-process.md]: `30 minutes`

File: 2_Operations/work-process.md (hunk 10 above)
Comment (dz0ny) [reply 2_Operations/work-process.md]: Preferably yes, unless that room is in use...

**Pull request merged**